packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -618,26 +616,26 @@ typedef HighlightAutoComplete = Function(
 /// Callback for clearing the search field.
 typedef ClearSearchField = Function(
   SearchControllerMixin controller, {
-  bool force,
+  bool? force,
```

Comment:
make this non-nullable with a default of false

Reply:
done